docs/Tutorial.md

```diff
@@ -155,18 +155,20 @@ export const PostList = (props) => (
 );
 ```
 
-When displaying the posts list, the browser now fetches related user records, and displays their name as a `<TextField>`.
+When displaying the posts list, the browser now fetches related user records, and displays their `name` as a `<TextField>`.
 
 ![reference posts in comment list](http://static.marmelab.com/admin-on-rest/reference_posts.png)
 
+**Tip**: Reference components always pass the data they fetch to a child component, which is responsible for displaying the data.
+
 ## Creation and Edition
 
 An admin interface is usually for more than seeing remote data - it's for editing and creating, too. Admin-on-REST provides `<Create>` and `<Edit>` components for that purpose. We'll add them to the `posts` resource:
 
 ```js
 // in src/posts.js
 import React from 'react';
-import { List, Edit, Create, ReferenceField, TextField, EditButton, DisabledInput, LongTextInput, ReferenceInput, TextInput } from 'admin-on-rest/lib/mui';
+import { List, Edit, Create, ReferenceField, TextField, EditButton, DisabledInput, LongTextInput, ReferenceInput SelectObjectInput, TextInput } from 'admin-on-rest/lib/mui';
```

Comment:
You can already import them individually. It's just more convenient to require whatever you need from a single file.